packages/core/content-type-builder/admin/src/components/ComponentCard/ComponentIcon/ComponentIcon.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+import { Flex } from '@strapi/design-system';
+
+const Wrapper = styled(Flex)`
+  border-radius: 50%;
+  color: ${({ theme, isActive }) => (isActive ? theme.colors.primary600 : theme.colors.neutral600)};
+  height: ${({ theme }) => theme.spaces[8]};
+  width: ${({ theme }) => theme.spaces[8]};
+
+  svg {
+    height: ${({ theme }) => theme.spaces[5]};
+    width: ${({ theme }) => theme.spaces[5]};
+  }
+`;
+
+export function ComponentIcon({ isActive }) {
+  return (
+    <Wrapper
+      alignItems="center"
+      background={isActive ? 'primary200' : 'neutral200'}
+      justifyContent="center"
+      isActive={isActive}
+    >
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
+        <path
+          d="M216.3 2c4.8-2.6 10.5-2.6 15.3 0L422.3 106c5.1 2.8 8.3 8.2 8.3 14s-3.2 11.2-8.3 14L231.7 238c-4.8 2.6-10.5 2.6-15.3 0L25.7 134c-5.1-2.8-8.3-8.2-8.3-14s3.2-11.2 8.3-14L216.3 2zM23.7 170l176 96c5.1 2.8 8.3 8.2 8.3 14V496c0 5.6-3 10.9-7.8 13.8s-10.9 3-15.8 .3L8.3 414C3.2 411.2 0 405.9 0 400V184c0-5.6 3-10.9 7.8-13.8s10.9-3 15.8-.3zm400.7 0c5-2.7 11-2.6 15.8 .3s7.8 8.1 7.8 13.8V400c0 5.9-3.2 11.2-8.3 14l-176 96c-5 2.7-11 2.6-15.8-.3s-7.8-8.1-7.8-13.8V280c0-5.9 3.2-11.2 8.3-14l176-96z"
```

Comment:
out of curiosity: we didn't want this 3d box as a DS icon I guess? 👀

Reply:
Which Icon are you referring to? I couldn't find any box ...

Reply:
ah sorry, I meant this one 
this path here is for this icon no? 👀

<img width="63" alt="image" src="https://user-images.githubusercontent.com/71838159/208704944-f3f4f382-8131-4889-a4a6-f6594afcaf3e.png">

Reply:
I think it doesn't exist, that's why I was wondering if it was on purpose not having it in the DS 😄
no blocker though, if we want to add it we can iterate on it!

Reply:
Right, get it now 🥸 I didn't ask if we would want to add it in the DS because I don't find it very reusable, but I'm happy to open this discussion if you think it is a good idea.

Reply:
let's wait and see if it's ever needed in another place 😊